src/engines/html5/html5.js

```diff
@@ -941,4 +941,89 @@ export default class Html5 extends FakeEventTarget implements IEngine {
     }
     return extended;
   }
+
+  enterPictureInPicture(): void {
+    try {
+      // Currently it's supported in chrome and in safari. So if we consider checking support before,
+      // we can use this flag to distinguish between the two. In the future we might need a different method.
+      // Second condition is because flow does not support this API yet
+      if (document.pictureInPictureEnabled && typeof this._el.requestPictureInPicture === 'function') {
+        this._el.requestPictureInPicture().catch(error => {
+          this.dispatchEvent(
+            new FakeEvent(
+              Html5EventType.ERROR,
+              new Error(Error.Severity.RECOVERABLE, Error.Category.PLAYER, Error.Code.PICTURE_IN_PICTURE_FAILURE, {
+                action: 'Enter picture in picture',
```

Comment:
why do we need the action property which is a free string?
why not to split error codes to `ENTER_PICTURE_IN_PICTURE_FAILED` & `EXIT_PICTURE_IN_PICTURE_FAILED`

Reply:
changed back to two errors.